plotly/utils.py

```diff
@@ -257,19 +262,21 @@ def encode_as_numpy(obj):
     @staticmethod
     def encode_as_datetime(obj):
         """Attempt to convert to utc-iso time string using datetime methods."""
-
-        # first we need to get this into utc
-        try:
-            obj = obj.astimezone(pytz.utc)
-        except ValueError:
-            # we'll get a value error if trying to convert with naive datetime
-            pass
-        except TypeError:
-            # pandas throws a typeerror here instead of a value error, it's OK
-            pass
-        except AttributeError:
-            # we'll get an attribute error if astimezone DNE
-            raise NotEncodable
+        # In PY36, isoformat() converts UTC
+        # datetime.datetime objs to UTC T04:00:00
```

Comment:
This is odd - why did this need to change?

Reply:
see above comment